secure-api-gateway-ob-uk-rcs-api/src/main/java/com/forgerock/sapi/gateway/ob/uk/rcs/api/dto/consent/decision/ConsentDecisionDeserialized.java

```diff
@@ -35,4 +35,5 @@ public class ConsentDecisionDeserialized {
     private String decision;
     private List<String> accountIds;
     private FRFinancialAccount debtorAccount;
+    private boolean debtorAccountProvided;
```

Comment:
Where is this flag being set?

Reply:
We not need anymore that flag, has been deleted in the latest commit.